Dockerfile

```diff
@@ -4,10 +4,10 @@
 # <NODE>
 ADD tools/build /stack/boilerplate
 
-ENV NODE_VERSION=6.10.1 \
-    NPM_VERSION=3.10.10
+ENV NODE_VERSION=8.9.4 \
+    NPM_VERSION=5.4.0
 
-RUN bash /stack/boilerplate/install.sh
+RUN sh /stack/boilerplate/install.sh
 
 ENV NODE_PATH=$NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules \
     PATH=$NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
```

Comment:
Is this still correct also for the alpine image?

Reply:
This needs to move to the debian branch only